doc/man/qvm.1

```diff
@@ -133,6 +133,9 @@ Perform a benchmark on <qubits> number of qubits (default is
 .IP "--benchmark-type <type>"
 The type of benchmark to run. Available options are "bell" (default),
 "qft", "hadamard", "qualcs", or "suite" (non-SDK).
+.IP "--single-user"
```

Comment:
I don't think this is a very informative name. Why not `--parallelize-multishot` since that is what it does.

Reply:
Because there are other possibilities, like not spinning up one thread per request.

Reply:
Collecting all those things under one config option doesn't give the user much option.

Reply:
Would it be worth it to expand on the possibilities in the docs/help output? A use-case or something would help bridge the gap between "I want parallel shots" and  "Single-user mode enables parallel shots (and other cool things)"

Reply:
> expand on the possibilities in the docs/help output?

I am trying to figure out multishot. Extending the EXAMPLES section in the man page could be cheap way to help people understand (beyond multishot too). Also, a collection of examples might help focus design discussions.